lava-slave-install.md

```diff
@@ -0,0 +1,107 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+Along with this file, you could see standalone-slave.yaml.example, example are based on it.
```

Comment:
prof d'anglais nit:
Along with this file, you could see standalone-slave.yaml.example file.  The rest of the document is based on that example